pom.xml

```diff
@@ -16,17 +16,17 @@
     <properties>
         <java.version>21</java.version>
         <start-class>uk.gov.companieshouse.pscdataapi.PscDataApiApplication</start-class>
-        <spring-boot-dependencies.version>3.3.1</spring-boot-dependencies.version>
-        <spring-boot-maven-plugin.version>3.3.1</spring-boot-maven-plugin.version>
+        <spring.boot.version>3.3.5</spring.boot.version>
```

Comment:
These can be 3.4.0 and the parent pom version can be 2.1.10

Reply:
done

Reply:
spring boot versions cannot go above the current as the config is incorrect in tests and breaks at higher version, as this needs to go out soon I've talked to Sam and he's okay with me not including the spring boot upgrade as it'll be done in future ticket